code/aes/AUTHORS.md

```diff
@@ -0,0 +1 @@
+This code was primarily written by Karthikeyan Bhargavan (INRIA).
```

Comment:
@karthikbhargavan can you find a phrasing that incorporates @mamonet's contributions?

Reply:
Done